[v2.10.0] Add docs about enable/disable functionality in clusterrepo (#1462)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

Airgap installations where Rancher is configured to use the packaged copy of Helm system charts ([`useBundledSystemChart=true`](../../../getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md#helm-chart-options-for-air-gap-installations)) will only refer to the [system-chart](https://github.com/rancher/system-charts) repository that comes bundled and will not be able to be refreshed or synced.

### Enable/Disable Helm Chart Repositories

> A new feature is added to enable and disable a Helm repository. A Helm repository is enabled by default.

Reviewer: ```suggestion
Rancher v2.10.0 adds the ability to enable and disable Helm repositories. Helm repositories are enabled by default.
```

To disable a chart repository:

> 1. Find the repository you want to disable, and click **⋮ > Edit YAML**.

Reviewer: We should add how to navigate to the place where repositories are listed

> 1. Set the `enabled` field under `Spec` to false.

Reviewer: ```suggestion
1. Set the **Enabled** field under **Spec** to **false**.
```

> 1. Click on **Save**.

Reviewer: ```suggestion
1. Click **Save**.
```

> 1. When you disable a repository, updates are disabled and new changes to the clusterRepo are not picked up.

Reviewer: ```suggestion
1. When you disable a repository, updates are disabled and new changes to the clusterRepo are not applied.
```

> To enabled a chart repository:

Reviewer: ```suggestion
To enabled a chart repository:

```

1. Find the repository you want to disable, and click **⋮ > Edit YAML**.

> 1. Set the `enabled` field under `Spec` to true.

Reviewer: ```suggestion
1. Set the **Enabled** field under **Spec** to **true**.
```

> 1. Click on **Save**.

Reviewer: ```suggestion
1. Click **Save**.
```
